libs/common/types.hpp

```diff
@@ -269,6 +281,8 @@ namespace iroha {
     };
   }
 
+  using namespace ed25519; // to use sig_t everywhere instead of ed25519::sig_t
```

Comment:
No `using namespace` in header files according to codestyle. It makes `namespace ed25519` defined earlier useless.